apps/condo/domains/property/components/PropertyAddressSearchInput.tsx

```diff
@@ -41,11 +41,20 @@ export const PropertyAddressSearchInput: React.FC<IAddressSearchInput> = (props)
 
     const searchAddress = useCallback(
         (query, skip) => {
+            const splitted_query = query.split(/\s|\.|,/).filter(e => e != '')
```

Comment:
It's not a good regular expression, as it do not filter double spaced

Reply:
> 

It does. It splits by all spaces and remove empty strings. So it doesn't matter how many spaces, dots and commas in search